src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/paymentnotification/PaymentNotificationIngestionActivityImpl.java

```diff
@@ -54,7 +54,7 @@ protected PaymentNotificationIngestionFlowFileResult handleRetrievedFiles(
       return csvService.readCsv(filePath,
           PaymentNotificationIngestionFlowFileDTO.class, (csvIterator, readerException) ->
               paymentNotificationProcessingService.processPaymentNotification(csvIterator, readerException,
-                  ingestionFlowFileDTO, workingDirectory));
+                  ingestionFlowFileDTO, workingDirectory), ingestionFlowFileDTO.getFileVersion());
```

Comment:
This flow has not been been rightly be configured with version on csv DTO (neither ingestionFlowFileDTO.getFileVersion() is probably set)
```suggestion
                  ingestionFlowFileDTO, workingDirectory), null);
```